### ISSUE TITLE
NVDA does not read incoming and outgoing chat messages in split view of Skype 8.51 and newer

### ISSUE THREAD

**Issue by k-kolev1985** [nvaccess/nvda#10138]
### Steps to reproduce:
1. Install Skype 8.51.0.86 or newer and launch it.
2. In the Skype's main window, with the TAB key, go to the "More options" button (the one after your Skype name) and activate it. A menu will appear.
3. From that menu, select and activate the command "Enable split view mode".
4. Initiate a new chat with someone.
5. Press NVDA+Control+F1 to make sure that the Skype module is loaded.
6. Exchange a few incoming and outgoing messages.

### Actual behavior:
NVDA stays silent when incoming and outgoing messages appear in the separate chat window. It may read incoming messages, but those come not from the current chat window, but from the system toast notifications for Skype.

### Expected behavior:
NVDA should read the content of incoming chat messages for the currently focused chat window and say "Message sent" for outgoing messages, as it does when split view is disabled.

### System configuration
- Operating system: Windows 10 Pro version 1903 (build 18362.295), 64-bit, in Bulgarian with all locale settings set to "Bulgarian".
- Skype version: 8.51.0.86.
- NVDA version: 2019.2, alpha-18420,a8e8db19.
- Processor: Intel Core i5-4670 at 3.40GHz.
- RAM Memory: 8.00GB.
- Graphics: MSI GeForce GTX-1050TI Gaming X 4G, 4096MB dedicated memory + 4065MB shared system memory, desktop resolution set to 1920x1080.
- Sound Card: Realtek ALC887 at Intel Lynx Point PCH - High Definition Audio Controller.

#### Does the issue still occur after restarting your PC?
Yes, it does.

#### Have you tried any other versions of NVDA? If so, please report their behaviors.
Only 2019.2 and the latest alpha build.

**Comment by Adriani90** [comment]
@k-kolev1985 I assume you tested in Skype for desktop? Or is this the skype UWP from windows store?

**Comment by Adriani90** [comment]
cc: @leonardder

**Comment by k-kolev1985** [comment]
@Adriani90 Yes, I've tested with Skype for desktop. The problem does not occur with Skype from the store.

**Comment by Adriani90** [comment]
@k-kolev1985 I tested in skype electron for desktop version 8.58, when I press enter after writing a message, NVDA reports that it has been sent. So this issue might be solved now. Could you test please?

**Comment by Adriani90** [comment]
Is this a duplicate of #8591?

**Comment by k-kolev1985** [comment]
@Adriani90 I've tested with Skype version 8.59.76.73 (insider build) and I can still reproduce the problem. Previously I was using v8.58 and I still could reproduce it. I don't know how it works for you in split window mode. And what about incoming chat messages? Does NVDA read them to you when the chat window in question is in focus?

And no, issue #8591 is not the same. That one affects single window mode as well and it is about the fact that instead of reading the content of the message sent, NVDA reads the literal text "message sent".

**Comment by Adriani90** [comment]
@k-kolev1985 are you still having the issue with incomming messages in Skype 8.94?

**Comment by k-kolev1985** [comment]
@Adriani90 Sometimes, but it is hard to pinpoint the circumstances under which it occurs. And it is probably more Skype's fault than NVDA's. So we might as well close the issue.

**Comment by Adriani90** [comment]
@k-kolev1985 I think it is hard to investigate this, especially when it occurs only sometimes. There are some requests in this repository about UIA events which should be tracked for specific regions of a window, implementing that will probably improve this as well. cc: @josephsl.

**Comment by Adriani90** [comment]
Skype retires completely in May 2025, I don't expect any accessibility update comming further. I am closing as abandoned.
https://support.microsoft.com/en-us/skype/skype-is-retiring-in-may-2025-what-you-need-to-know-2a7d2501-427f-485e-8be0-2068a9f90472

If you have this issue in Skype for business which is still maintained by Microsoft, then please comment and we can reopen.
